releasenotes/notes/ec2-exclude-tags-c2b92f1716ef6bee.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Adding a configuration option, ``exclude_ec2_tags``, to exclude EC2 instance tags from being converted into host
```

Comment:
```suggestion
    Adding a configuration option, `exclude_ec2_tags`, to exclude EC2 instance tags from being converted into host
```

Reply:
The release notes are exported as RST in the changelog, we need the double backtick.